plutus-core/cost-model/data/builtinCostModelA.json

```diff
@@ -1100,5 +1113,18 @@
             "arguments": 1,
             "type": "constant_cost"
         }
+    },
+    "ripemd_160": {
+        "cpu": {
+            "arguments": {
+                "intercept": 1964219,
+                "slope": 24520
+            },
+            "type": "linear_in_x"
+        },
+        "memory": {
+            "arguments": 3,
+            "type": "constant_cost"
+        }
```

Comment:
VERY IMPORTANT!

This has 2 entries for RIPEMD (with different values actually), which is dangerous considering it is JSON: based on the specific implementation, a json library silently picks the first occurrence or the last occurence. Aeson I think picks the first one.

Reply:
Thank you.  I hope that's fixed now.  I added the new costing function but forgot to delete the preliminary one (which was a duplicate of the one for keccak_256).  The old one was also in the wrong place, which might not have been so good.